CONTRIBUTING.md

```diff
@@ -8,6 +8,21 @@ We gratefully welcome improvements to documentation as well as to code.
 
 ## Getting Started
 
+### Pre-requisites
+
+* [Go](https://golang.org/doc/install).
+* Docker version 23.0.0 or greater.
```

Comment:
i also would recommend using kind for local development, i think that's a requirement in there?

Reply:
though i guess this is mentioned below...

Reply:
That's not strictly speaking a requirement. If you run `prepare-e2e` for example, kind will be installed automatically, so you only need docker. I think I'd prefer to add a separate section on how to run various commands in your own cluster, as it's more involved.